src/org/pahappa/systems/ticketing/views/TicketView.java

```diff
@@ -1,17 +1,21 @@
 package org.pahappa.systems.ticketing.views;
 
-import org.pahappa.systems.ticketing.services.TicketService;
+import org.pahappa.systems.ticketing.constants.*;
+import org.pahappa.systems.ticketing.models.*;
+// import org.pahappa.systems.ticketing.services.TicketService;
 import org.pahappa.systems.ticketing.services.impl.TicketServiceImpl;
 
 import java.util.Scanner;
 
 public class TicketView implements BaseTicketView {
 
-    private final TicketService ticketService;
+    // private final TicketService ticketService;
     private final Scanner scanner;
```

Comment:
Since the starter code previously had the `ticketService` declared kindly maintain that.
`private final TicketService ticketService;`